app/assets/javascripts/student_profile/take_notes.jsx

```diff
@@ -97,6 +98,45 @@ export default React.createClass({
     return (noteInProgressType === null || !this.isValidAttachmentUrls());
   },
 
+  noteText() {
+    // In most cases, display the text the user has entered into the controlled
+    // text input, passed in as props. When the user is a counselor entering a
+    // High School Transition Note, show an initial prompt.
+
+    const {noteInProgressText, noteInProgressType} = this.props;
+
+    if (noteInProgressType !== 307) return noteInProgressText;
+
+    if (noteInProgressText !== '') return noteInProgressText;
+
+    // Prompt for High School Transition Note
+    return `What are this student's strengths?
+——————————
+
+
+What is this student's involvement in the school community like?
+——————————
+
+
+How does this student relate to their peers?
+——————————
+
+
+Is this student receiving Social Services and if so, what is the name and contact info of their social worker?
```

Comment:
Les's feedback on these initial prompts: 

> Questions 2 & 3 are NOT too similar, and I think they could be left as is.

> Questions 4 & 5 on the other hand could be merged into a combined question that I will think about wording.

Les also suggested merging the first and the last question into "Any additional strengths, comments, or good things to know about this student?" I wonder if there's some value into explicitly starting off the prompts with a positively-framed question.